app/Models/User.php

```diff
@@ -6,21 +6,23 @@
 
 class User extends Authenticatable
 {
-    /**
-     * The attributes that are mass assignable.
-     *
-     * @var array
-     */
+
+    protected $table = 'users';
+
     protected $fillable = [
-        'name', 'email', 'password',
+        'name', 'email', 'password', 'status', 'token', 'last_restore'
     ];
 
-    /**
-     * The attributes that should be hidden for arrays.
-     *
-     * @var array
-     */
     protected $hidden = [
-        'password', 'remember_token',
+        'password', 'remember_token'
     ];
+
+    const STATUS_ACTIVE = 'active';
+
```

Comment:
please, remove lines between same-meaning constants.
Separate them with new line only if they related to different fields.